CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+ * [`added`] internal tests to the repository.
```

Comment:
I don't think that the tests are relevant for the users of embedded-sht which the CHANGELOG is aimed at, since it doesn't change anything for them.